index.js

```diff
@@ -9,13 +9,14 @@ var express = require("express"),
     labelModel = require('./lib/label-model'),
```

Comment:
Nit: can we make all these imports `const`? I'm not sure it'd help with the name collision you found (yikes), but hopefully it's better than nothing.

Reply:
Done, and sorted them, which helped me realize we include github twice! :O